src/assets/images/icons/header/game.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Please, use some minification tool, for this and other images. You can find such tool in google or here https://css-tricks.com/tools-for-optimizing-svg/

Reply:
Done